EstateAdministrationUI/Areas/Estate/Controllers/HomeController.cs

```diff
@@ -203,34 +261,59 @@ public IActionResult Index()
             return this.View();
         }
 
+        [HttpGet]
+        public async Task<IActionResult> MakeMerchantDeposit([FromQuery] Guid merchantId,
+                                                             [FromQuery] String merchantName,
+                                                             CancellationToken cancellationToken)
+        {
+            MakeMerchantDepositViewModel viewModel = new MakeMerchantDepositViewModel();
+            viewModel.MerchantId = merchantId.ToString();
+            viewModel.MerchantName = merchantName;
+
+            return this.View("MakeMerchantDeposit", viewModel);
+        }
+
         /// <summary>
-        /// Manages the estate.
+        /// Creates the merchant.
         /// </summary>
+        /// <param name="viewModel">The view model.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
         /// <returns></returns>
-        [HttpGet]
-        public async Task<IActionResult> GetEstate(CancellationToken cancellationToken)
+        [HttpPost]
+        public async Task<IActionResult> MakeMerchantDeposit(MakeMerchantDepositViewModel viewModel,
+                                                             CancellationToken cancellationToken)
         {
-            String accessToken = await this.HttpContext.GetTokenAsync("access_token");
+            // Validate the model
+            if (this.ValidateModel(viewModel))
+            {
+                String accessToken = await this.HttpContext.GetTokenAsync("access_token");
 
-            EstateModel estateDetails = await this.ApiClient.GetEstate(accessToken, this.User.Identity as ClaimsIdentity, cancellationToken);
+                MakeMerchantDepositModel makeMerchantDepositModel = this.ViewModelFactory.ConvertFrom(viewModel);
 
-            return this.View("EstateDetails", this.ViewModelFactory.ConvertFrom(estateDetails));
+                // All good with model, call the client to create the golf club
+                MakeMerchantDepositResponseModel makeMerchantDepositResponseModel =
```

Comment:
Codacy found an issue: [Remove the unused local variable 'makeMerchantDepositResponseModel'.](https://app.codacy.com/manual/stuart_ferguson1/EstateAdministrationUI/pullRequest?prid=5745005)